### ISSUE TITLE
Scroll Bar?

### ISSUE THREAD

**Issue by Ghostttie** [npwd-community/npwd_qb_mail#14]
As you can see from the image below when receiving an email which contains a lot of text you're actually unable to view most of it due to not being able to scroll down in the email, if a scroll feature of some sorts could be implemented that would honestly be golden.
![image](https://user-images.githubusercontent.com/64477318/190708344-e10f7487-9e65-462a-a546-d729ab7ede25.png)

Keep up the good work!

**Comment by CptNoHand** [comment]
Yes, i agree. I had the same few days ago and forgot to post it here.

**Comment by mockdot** [comment]
This fixes it #20
